worldeditadditions_commands/commands/uasparse.lua

```diff
@@ -0,0 +1,29 @@
+local wea_c = worldeditadditions_core
+-- local Vector3 = wea_c.Vector3
+
+-- ██    ██   █████   ███████  ██████    █████   ██████   ███████  ███████ 
+-- ██    ██  ██   ██  ██       ██   ██  ██   ██  ██   ██  ██       ██      
+-- ██    ██  ███████  ███████  ██████   ███████  ██████   ███████  █████   
+-- ██    ██  ██   ██       ██  ██       ██   ██  ██   ██       ██  ██      
+--  ██████   ██   ██  ███████  ██       ██   ██  ██   ██  ███████  ███████ 
+
+worldeditadditions_core.register_command("uasparse", {
+	params = "<unified axis syntax>",
+	description = "Returns min and max vectors for given inputs",

```

Comment:
This is likely a debug command, no?
```suggestion
	description = "Debug command. Returns min and max vectors for given inputs",
```